test/HostActivationTests/GivenThatICareAboutPortableAppActivation.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using Xunit;
+using FluentAssertions;
+using Microsoft.DotNet.CoreSetup.Test;
+
+namespace Microsoft.DotNet.CoreSetup.Test.HostActivation.PortableApp
+{
+    public class GivenThatICareAboutPortableAppActivation
+    {
+        private static TestProjectFixture PortableTestProjectFixture { get; set; }
+		private static string ExeExtension { get; set; }
+
+		static GivenThatICareAboutPortableAppActivation()
+        {
+            ExeExtension = RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
```

Comment:
Should we put a GetExeExtension method in RuntimeInformation instead of having this code duplicated in a few different places?